src/test/java/picard/util/TestNGUtil.java

```diff
@@ -96,6 +100,15 @@ public static void assertEqualDoubleArrays(final double[] lhs, final double[] rh
         }
     }
 
+    public static void assertEquals(final byte[] actual, final byte[] expected) {
```

Comment:
TestNG already has `Assert.assertEquals(byte[] actual, byte[] expected)`, is there a reason why that wouldn't work here?

Reply:
yes, IINM TestNG doesn't show you the offending difference....which is less helpful for debugging.

Reply:
Pretty sure that's not true; looking at the code (`org.testng.Assert`) I see ```  public static final String ARRAY_MISMATCH_TEMPLATE = "arrays differ firstly at element [%d]; "
      + "expected value is <%s> but was <%s>. %s";
```

Reply:
hmm. I keep getting pointed at `Assert.assertEquals(Object, Object)`

Reply:
are you seeing that signature in intellij or in a website?

Reply:
Sorry, I was looking at the copy that htsjdk uses. If you update testng to the same version that htsjdk uses, then you should have the byte compare assert that prints out the difference.
`build.gradle`:
```
    testCompile 'org.testng:testng:6.14.3'
```